src/index.html

```diff
@@ -7,8 +7,92 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="header__link"
+        href="#home"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo Logo"
+        >
+      </a>
+
+      <nav class="header__nav">
+        <ul class="nav_list">
+          <li class="nav_list-item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+             Apple
+            </a>
+          </li>
+          <li class="nav_list-item">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
          </li>
          
          <li class="nav_list-item">
```